doc/release-notes.md

```diff
@@ -47,44 +94,41 @@ originally received.
-  process table; `zcashd-wallet-tool` is specifically provided to avoid this
-  problem.
-- `z_getnewaccount` This API allows for creation of new BIP 44 / ZIP 32
-  accounts using HD derivation from the wallet's mnemonic seed. Each account represents a
-  separate spending authority and source of funds. A single account may contain funds in
-  the Sapling and Orchard shielded pools, as well as funds held in transparent addresses.
-- `z_getaddressforaccount` This API allows for creation of diversified unified
-  addresses under a single account. Each call to this API will, by default, create
-  a new diversified unified address containing p2pkh, Sapling, and Orchard receivers.
-  Additional arguments to this API may be provided to request the address to be created
-  with a user-specified set of receiver types and diversifier index.
-- `z_getbalanceforaccount` This API makes it possible to obtain balance information
-  on a per-account basis.
-- `z_getbalanceforviewingkey` This newly created API allows a user to obtain
-  balance information for funds visible to a Sapling or Unified full viewing key; if a
-  Sprout viewing key is provided, this method allows retrieval of the balance only in the
-  case that the wallet controls the corresponding spending key.  This API has been added
-  to supplement (and largely supplant) `z_getbalance`. Querying for balance by a single
+- `walletconfirmbackup` This newly created API checks a provided emergency recovery phrase 
+  against the wallet's emergency recovery phrase; if the phrases match then it updates the
+  wallet state to allow the generation of new addresses.  This backup confirmation
+  workflow can be disabled by starting zcashd with `-requirewalletbackup=false` but this
+  is not recommended unless you know what you're doing (and have otherwise backed up the
+  wallet's recovery phrase anyway).  For security reasons, this RPC method is not intended
+  for use via zcash-cli but is provided to enable `zcashd-wallet-tool` and other
+  third-party wallet interfaces to satisfy the backup confirmation requirement. Use of the
+  `walletconfirmbackup` API via zcash-cli would risk that the recovery phrase being
+  confirmed might be leaked via the user's shell history or the system process table;
+  `zcashd-wallet-tool` is specifically provided to avoid this problem.
+- `z_getnewaccount` This API allows for creation of new BIP 44 / ZIP 32 accounts using HD
+  derivation from the wallet's mnemonic seed. Each account represents a separate spending
+  authority and source of funds. A single account may contain funds in the Sapling and
+  Orchard shielded pools, as well as funds held in transparent addresses.
+- `z_getaddressforaccount` This API allows for creation of diversified unified addresses 
+  under a single account. Each call to this API will, by default, create a new diversified
+  unified address containing p2pkh, Sapling, and Orchard receivers.  Additional arguments
```

Comment:
Should "p2pkh" be "transparent" for clarity?

Reply:
```suggestion
  unified address containing transparent p2pkh, Sapling, and Orchard receivers.  Additional arguments
```